experimental/dds/tree2/src/index.ts

```diff
@@ -159,7 +158,7 @@ export {
 	cursorFromContextualData,
 	AllowedTypes,
 	TreeNodeSchema as FlexTreeNodeSchema,
-	TreeSchema,
+	FlexTreeSchema as TreeSchema,
```

Comment:
Since the above export is `FlexTreeNodeSchema`, does this want to remain non-aliased to match?